manager/controllers/utils/utils.go

```diff
@@ -134,6 +134,29 @@ func Intersection(set1, set2 []string) []string {
 }
 
 const FybrikAppUUID = "app.fybrik.io/app-uuid"
+const FybrikModuleNamespace = "app.fybrik.io/modules-namespace"
+
+// GetFybrikModuleNamespaceFromLabels returns the moduleNamespace passed to the resource in its labels
+func GetFybrikModuleNamespaceFromLabels(labels map[string]string) string {
```

Comment:
This function is redundant. The relevant code can be inserted into GetFybrikModuleNamespace.

Reply:
Fixed. Thanks